### PR TITLE
V1.0/s2/feature/frontend/samples/create

### PR DESCRIPTION
**Pull request by louMon**
Herbert, can we check the code, please?

### CODE REVIEW

File: app/assets/javascripts/pages/samples.js [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
-      tabcontent = document.getElementsByClassName("tabcontent");
-      for (i = 0; i < tabcontent.length; i++) {
-          tabcontent[i].style.display = "none";
-      }
-      tablinks = document.getElementsByClassName("tablinks");
-      for (i = 0; i < tablinks.length; i++) {
-          tablinks[i].className = tablinks[i].className.replace(" active", "");
+
+  $(function(){
+    var $accordions = $(".accordion");
```
Comment (Orange-walker-95) [inline app/assets/javascripts/pages/samples.js]: Use only one $(function() {} ) per file.

File: app/views/samples/_form.html.erb [hunk 2]
```diff
@@ -1,18 +1,288 @@
...
+  <div class="col-md-6">
+      <div class="col-md-10">
+          <label class="select-float-label">Razón Social</label>
+          <input id="txtRazonSocial" name="txtRazonSocial" type="text" class="form-control required" maxlength="80" required>
+      </div>
+  </div>
+  <div class="col-md-4">
+      <div class="col-md-10">
+          <label class="select-float-label">RUC</label>
+          <input id="txtRuc" name="txtRuc" type="text" class="form-control required only-numbers"  maxlength="11" >
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: The should be required

File: app/views/samples/_form.html.erb [hunk 3]
```diff
@@ -1,18 +1,288 @@
-<p>Click on the buttons inside the tabbed menu:</p>
 
-<div class="tab">
-  <button class="tablinks" onclick="openData(event, 'Cliente')">Cliente</button>
-  <button class="tablinks" onclick="openData(event, 'MuestrasGeneral')">MuestrasGeneral</button>
+<div class="form-group row">
+  <div class="col-md-12">
+      <div class="p-xxs border-bottom col-md-12">
+          <h4><strong><i class="fa fa-user" aria-hidden="true"></i> Cliente</strong></h4>
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: what ? Use a more descriptive name

File: app/views/samples/_form.html.erb [hunk 4]
```diff
@@ -1,18 +1,288 @@
...
+          </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-8">
+                <label class="select-float-label">Tipo de Muestra</label>
+                <select class="form-control"><option>Selecciona una opcion</option> </select>
+            </div>
+            <div class="col-md-4">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: You should use multiples lines.

File: app/views/samples/_form.html.erb [hunk 5]
```diff
@@ -1,18 +1,288 @@
...
+            </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-6">
+              <label class="select-float-label">Valor</label>
+              <input id="txtMuestra" name="txtMuestra" type="text" class="form-control required" maxlength="80" required>
+            </div>
+            <div class="col-md-6">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 6]
```diff
@@ -1,18 +1,288 @@
...
+          </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-8">
+                <label class="select-float-label">Categoría</label>
+                <select class="form-control"><option>Selecciona una opcion</option> </select>
+            </div>
+            <div class="col-md-2">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 7]
```diff
@@ -1,18 +1,288 @@
...
+            </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-6">
+              <label class="select-float-label">Valor</label>
+              <input id="txtMuestra" name="txtMuestra" type="text" class="form-control required" maxlength="80" required>
+            </div>
+            <div class="col-md-6">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 8]
```diff
@@ -1,18 +1,288 @@
...
+                  Observaciones
+                </label>
+            </div>
+        </div>
+        <div class="col-md-4">
+          <label class="checkbox text-center">
+              <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 9]
```diff
@@ -1,18 +1,288 @@
...
+          </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-8">
+                <label class="select-float-label">Tipo de Muestra</label>
+                <select class="form-control"><option>Selecciona una opcion</option> </select>
+            </div>
+            <div class="col-md-4">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 10]
```diff
@@ -1,18 +1,288 @@
...
+            </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-6">
+              <label class="select-float-label">Valor</label>
+              <input id="txtMuestra" name="txtMuestra" type="text" class="form-control required" maxlength="80" required>
+            </div>
+            <div class="col-md-6">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 11]
```diff
@@ -1,18 +1,288 @@
...
+          </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-8">
+                <label class="select-float-label">Categoría</label>
+                <select class="form-control"><option>Selecciona una opcion</option> </select>
+            </div>
+            <div class="col-md-2">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 12]
```diff
@@ -1,18 +1,288 @@
...
+            </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-6">
+              <label class="select-float-label">Valor</label>
+              <input id="txtMuestra" name="txtMuestra" type="text" class="form-control required" maxlength="80" required>
+            </div>
+            <div class="col-md-6">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 13]
```diff
@@ -1,18 +1,288 @@
...
+                  Observaciones
+                </label>
+            </div>
+        </div>
+        <div class="col-md-4">
+          <label class="checkbox text-center">
+              <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb]: Same as above.

File: app/views/samples/_form.html.erb [hunk 14]
```diff
@@ -1,18 +1,288 @@
...
+
+<div class="sical-form-buttons">
+  <div class="text-center">
+    <%= link_to "Cancelar", samples_path, class: "btn btn-primary sical-btn-cancel" %>
+    <button  class="btn btn-fill btn-default">Guardar</button>
+  </div>
+</div>  
+
+<script>
+  var acc = document.getElementsByClassName("accordion");
```
Comment (Orange-walker-95) [inline app/views/samples/_form.html.erb:316]: Don't use Js code, instead use jQuery and this should be in a js file.

File: app/views/samples/show.html.erb [hunk 15]
```diff
@@ -1 +1,292 @@
-@sample.description
\ No newline at end of file
+
+<div class="form-group row">
+  <div class="col-md-12">
+      <div class="p-xxs border-bottom col-md-12">
+          <h4><strong><i class="fa fa-user" aria-hidden="true"></i> Cliente</strong></h4>
```
Comment (Orange-walker-95) [inline app/views/samples/show.html.erb]: Use a more descriptive name for 'p-xxs' or deleted if you don't use it.

File: app/views/samples/show.html.erb [hunk 16]
```diff
@@ -1 +1,292 @@
...
+<div class="sical-form-buttons">
+  <div class="text-center">
+    <%= link_to "Cancelar", samples_path, class: "btn btn-primary sical-btn-cancel" %>
+    <button  class="btn btn-fill btn-default">Guardar</button>
+  </div>
+</div>  
+
+<script>
+  var $accordions = $(".accordion");
+  var i;
```
Comment (Orange-walker-95) [inline app/views/samples/show.html.erb]: This code should be in a separate js file.

File: app/views/samples/show.html.erb [hunk 17]
```diff
@@ -1 +1,292 @@
...
+          </div>
+        </div>
+        <div class="col-md-6 row">
+            <div class="col-md-8">
+                <label class="select-float-label">Tipo de Muestra</label>
+                <select class="form-control"><option>Selecciona una opcion</option> </select>
+            </div>
+            <div class="col-md-4">
+              <label class="checkbox text-center">
+                  <span class="icons"><span class="first-icon fa fa-square"></span><span class="second-icon fa fa-check-square "></span></span><input data-toggle="checkbox" value="" type="checkbox">
```
Comment (Orange-walker-95) [inline app/views/samples/show.html.erb]: Please check this file, it has the same mistakes as the samples/_form

File: app/assets/javascripts/pages/samples.js (hunk 1 above)
Comment (louMon) [reply app/assets/javascripts/pages/samples.js]: ok

File: app/views/samples/_form.html.erb (hunk 2 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 4 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 5 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 6 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 7 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 8 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 9 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 10 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 11 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 12 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/_form.html.erb (hunk 13 above)
Comment (louMon) [reply app/views/samples/_form.html.erb]: ok

File: app/views/samples/show.html.erb (hunk 15 above)
Comment (louMon) [reply app/views/samples/show.html.erb]: ok

File: app/views/samples/show.html.erb (hunk 17 above)
Comment (louMon) [reply app/views/samples/show.html.erb]: ok

**Pull request merged**